### PR TITLE
new recipe: qr_mumps

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: Q/qr_mumps/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+cd build/
+cmake .. -DARITH="d;s;c;z" -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$prefix -DQRM_ORDERING_AMD=OFF -DQRM_ORDERING_METIS=ON -DQRM_ORDERING_SCOTCH=OFF -DQRM_WITH_STARPU=OFF -DQRM_WITH_CUDA=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+make -j${nproc}
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_gfortran_versions(platforms)
```
Comment (giordano) [inline Q/qr_mumps/build_tarballs.jl:28]: Maybe
```suggestion
platforms = expand_gfortran_versions(supported_platforms())
```
?

Comment (amontoison) [reply Q/qr_mumps/build_tarballs.jl]: How can you answer so fast ? The time that I update my commit you already did the suggestion. :1st_place_medal:

**Comment by giordano** [comment]
Honestly, I don't have good recommendations besides disabling the failing platforms.  Especially for libgfortran3 I don't see easy solutions

**Comment by amontoison** [comment]
Ok, let's disable all failing platforms with libgfortran3.
Do you have an idea where the problem comes from with Windows + libgfortran 4/5 ?

File: Q/qr_mumps/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+cd build/
+cmake .. -DARITH="d;s;c;z" -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$prefix -DQRM_ORDERING_AMD=OFF -DQRM_ORDERING_METIS=OFF -DQRM_ORDERING_SCOTCH=OFF -DQRM_WITH_STARPU=OFF -DQRM_WITH_CUDA=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+make -j${nproc}
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_gfortran_versions(supported_platforms())
```
Comment (giordano) [inline Q/qr_mumps/build_tarballs.jl:28]: > Ok, let's disable all failing platforms with libgfortran3.

```suggestion
platforms = expand_gfortran_versions(supported_platforms())
filter!(p -> libgfortran_version(p) != v"3", platforms)
```

**Comment by amontoison** [comment]
I have a meeting with the main developer of `qr_mumps` this tuesday. I will see with him the errors on Windows. I will update the PR after.

**Comment by abuttari** [comment]
Hello, a new release is available which fixes this issue. It can be downloaded here:
http://buttari.perso.enseeiht.fr/qr_mumps/releases/qr_mumps-3.0.1.tgz

**Comment by giordano** [comment]
Wow, it worked, thank you so much!

**Comment by abuttari** [comment]
Awesome! thank you guys for pointing out the problem

**Pull request merged**